certs/production/montagu.vaccineimpact.org.crt

```diff
@@ -1,110 +1,39 @@
 -----BEGIN CERTIFICATE-----
-MIIFVzCCBD+gAwIBAgIJAN/wllrH5ZvyMA0GCSqGSIb3DQEBCwUAMIG0MQswCQYD
```

Comment:
Why did this have 3 certificates before, and now just 1?

Reply:
Very good catch. It turns out the thing I was using to inspect the certificate file was only printing out the first certificate in the bundle, leading me to think it wasn't a bundle. I've improved that now, and have (I think) properly created a new bundle.